python/nano/tutorial/notebook/training/pytorch-lightning/pytorch_lightning_cv_data_pipeline.ipynb

```diff
@@ -0,0 +1,245 @@
+    "        pred = torch.argmax(output, dim=1)\n",
+    "        acc = torch.sum(y == pred).item() / (len(y) * 1.0)\n",
+    "        metrics = {'test_acc': acc, 'test_loss': loss}\n",
+    "        self.log_dict(metrics)\n",
+    "\n",
+    "    def configure_optimizers(self):\n",
+    "        return torch.optim.SGD(self.parameters(), lr=0.002, momentum=0.9, weight_decay=5e-4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = MyLightningModule()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; _The definition of_ `MyLightningModule` _can be found in the_ [runnable example](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/pytorch-lightning/pytorch_lightning_cv_data_pipeline.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To finetune the model on OxfordIIITPet dataset, we need to create required train/validate dataloaders. To accelerate this data processing pipeline, you could simply **import BigDL-Nano** `transforms` **and** `datasets` **to replace** `torchvision.transforms` **and** `torchvision.datasets`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# from torchvision import transforms\n",
+    "# from torchvision.datasets import OxfordIIITPet\n",
+    "from bigdl.nano.pytorch.vision import transforms\n",
+    "from bigdl.nano.pytorch.vision.datasets import OxfordIIITPet\n",
+    "\n",
+    "# Data processing steps are the same as using torchvision\n",
+    "train_transform = transforms.Compose([transforms.Resize(256),\n",
+    "                                        transforms.RandomCrop(224),\n",
+    "                                        transforms.RandomHorizontalFlip(),\n",
+    "                                        transforms.ColorJitter(brightness=.5, hue=.3),\n",
+    "                                        transforms.ToTensor(),\n",
+    "                                        transforms.Normalize([0.485, 0.456, 0.406],\n",
+    "                                                            [0.229, 0.224, 0.225])])\n",
+    "val_transform = transforms.Compose([transforms.Resize(256),\n",
+    "                                    transforms.CenterCrop(224),\n",
+    "                                    transforms.ToTensor(),\n",
+    "                                    transforms.Normalize([0.485, 0.456, 0.406],\n",
+    "                                                            [0.229, 0.224, 0.225])])\n",
+    "\n",
+    "train_dataset = OxfordIIITPet(root=\"/tmp/data\", transform=train_transform, download=True)\n",
+    "val_dataset = OxfordIIITPet(root=\"/tmp/data\", transform=val_transform)\n",
+    "\n",
+    "# obtain training indices that will be used for validation\n",
```

Comment:
do we need to show so many details?

Reply:
Have revised, use a function to hide some details.